source/_components/geniushub.markdown

```diff
@@ -47,7 +47,22 @@ Each such entity will report back its primary state; in addition, `assigned_zone
 
 There are three `Sensor` entities that will indicate the number of **Errors**, **Warnings** and **Information** issues.
 
-Each such entity has a state attribute that will contain a list of any such issues. For example, `error_list`.
+Each such entity has a state attribute that will contain a list of any such issues which can be used in automations, etc. For example:
+
+```yaml
```

Comment:
This block contains a jinja template and needs to be escaped using `{% raw %}`

Reply:
Yeah - I dunno why I did that - trying to put it back now, but deploy check is fighting me!